### PR TITLE
blog: November newsletter

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: _posts/2017-11-30-foreman-community-newsletter---november-2017.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+show, either via YouTube Live chat, or via our IRC channel
+(Freenode/#theforeman).
+
+#### Community Demos / Deep Dives
+
+* [Community Demo - 16th Nov](https://www.youtube.com/watch?v=QHzNIFjMpTM)
+
+We had a busy demo with changes all around - the installer, smart proxies, UI.
+This included two introduction to new proposals - [Foreman telemetry proposal]
+and discourse.
```
Comment (GregSutcliffe) [inline _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: Capital D for Discourse :P

**Comment by GregSutcliffe** [comment]
Looks good, I added a Discourse paragraph. I guess 1.16 will be out today, so lets wait a little bit before merging this.

File: _posts/2017-11-30-foreman-community-newsletter---november-2017.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+about his reasoning [here][discourse_users], or watch the
+[AMA](https://www.youtube.com/watch?v=IdV6US_d4-U).
+
+The debate is not closed yet, so please do join in, either on the list
+thread above, or on the [migration poll][migration_poll] if you have views
+on this. At this time, the likely outcome is that we *will* move to
+Discourse in the near future, so watch out for more announcements about
+that in the coming weeks.
+
+### Katello 3.5 RC2
```
Comment (ekohl) [inline _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: I think that @akofink is working on the final release. Also note that @dLobatog is working on building the foreman 1.16.0 release. might be good to consider that too.

Comment (akofink) [reply _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: Yes, it's quite impossible (afaik) to build plugin releases (i.e. Katello) without having built Foreman first; however, out goal is to release 3.5 GA as quickly as possible after Foreman's GA.

**Comment by orrabin** [comment]
@GregSutcliffe, @ekohl I added 1.16 and a comment about katello 3.5 upcoming release.

File: _posts/2017-11-30-foreman-community-newsletter---november-2017.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+### Community lead
+
+Greg, our community lead, will be taking time off for the next 5 months.
+He's still around on irc and the mailing lists but not as active as usual.
+Until the beginning of May please feel free to contact Ori Rabin,
+orabin on irc.
+
+### New Sponsors?
+
+Fastly, our new sponsor has provided us with a [cdn] for the website.
```
Comment (GregSutcliffe) [inline _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: Was `[cdn]` meant to be a link?

File: _posts/2017-11-30-foreman-community-newsletter---november-2017.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+Also, new jenkins slaves from NETWAYS, thanks for that!
+
+### Upcoming Events
+
+* Community Demo - 7th December  (URL to be announced)
```
Comment (GregSutcliffe) [inline _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: Add Walden's UX demo?

**Comment by GregSutcliffe** [comment]
Two small comments, then +1 @orrabin

File: _posts/2017-11-30-foreman-community-newsletter---november-2017.md (hunk 3 above)
Comment (ekohl) [reply _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: This is actually a link. If the names match this is sufficient. So you could rename k_issues at the bottom to ```Katello bug tracker``` and simplify that link if you want.

Comment (orrabin) [reply _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: It was but I can change it.

**Comment by orrabin** [comment]
@GregSutcliffe I removed the link from cdn, I don't think we actually need it.
I also added a link to the demo since it's next week and I have the link, and the UX review that I missed.

**Comment by orrabin** [comment]
@GregSutcliffe approved this before leaving, no more comments were added so this can be merged.

File: _posts/2017-11-30-foreman-community-newsletter---november-2017.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+any issues you find to the [Katello bug tracker][k_issues]!
+
+### Community lead
+
+Greg, our community lead, will be taking time off for the next 5 months.
+He's still around on irc and the mailing lists but not as active as usual.
+Until the beginning of May please feel free to contact Ori Rabin,
+orabin on irc.
+
+### New Sponsors?
```
Comment (tbrisker) [inline _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: I don't think you need the `?` here.

File: _posts/2017-11-30-foreman-community-newsletter---november-2017.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+Fastly, our new sponsor has provided us with a new CDN for the website.
+
```
Comment (tbrisker) [inline _posts/2017-11-30-foreman-community-newsletter---november-2017.md]: extra blank line

**Review by tbrisker — CHANGES_REQUESTED** [review]
a couple of tiny comments

**Comment by orrabin** [comment]
@tbrisker thanks, fixed.

**Review by tbrisker — APPROVED** [review]

**Comment by tbrisker** [comment]
Thanks @orrabin !

**Pull request merged**